extensions/graphite-emitter/README.md

```diff
@@ -0,0 +1,69 @@
+## introduction
+
+This extension emits some selected druid metrics to a graphite carbon server.
+Events are sent after been [pickled](http://graphite.readthedocs.org/en/latest/feeding-carbon.html#the-pickle-protocol) , the size of the batch is configurable.
+Since Graphite doesn't support alerting, alerts will be redirect to warning logs by default. 
+
+ Metrics are organized using the following schema:
+`<namespacePrefix>.[<druid service name>].[<druid hostname>].<druid metrics dimensions>.<druid metrics name>`
```

Comment:
I mean if the druid service is running as a container, does such a schema still make sense?